### ISSUE TITLE
Can you make Hardcoded Characters get offsets

### ISSUE THREAD

**Issue by Hunter13n** [ShadowMario/FNF-PsychEngine#2599]
### What feature do you want to get added on the base engine?

So I made all my characters hardcoded an I can offset them, I can only change there positions but can’t scroll through there animations in character editior, can you make it where hardcoded character can also get supported it would help a lot of people.

**Comment by PersonNanoexe** [comment]
why would you hardcode it when using the character editor is like 57x easier

**Comment by Hunter13n** [comment]
i hardcode them because you make them playable characters instead of making 2 different json something like pico an pico-player, just make them a player an opponet with no json files

**Comment by Ninjatintin** [comment]
Just make two json files it's not that hard

**Comment by Hunter13n** [comment]
> Just make two json files it's not that hard

Yeah but then you would have to scroll through an look for 'matt' an 'bf-tankman'

**Comment by PersonNanoexe** [comment]
> Yeah but then you would have to scroll through an look for 'matt' an 'bf-tankman'

Mate it's not that difficult. It's actually a lot easier to make the jsons in the editor than hardcoding them. Trust me.
